lca-main.yaml

```diff
@@ -44,6 +44,15 @@ Parameters:
       Processor Lambda. Enabling Amazon Chime SDK Call Analytics also allows you to also use Voice Analytics, 
       such as Voice Tone Analysis and Speaker Search.
 
+  CustomVoiceConnectorId:
+    Type: String
+    Default: ''
+    MaxLength: 50
+    Description: >
+      (Optional) Id of an existing Amazon Chime SDK Voice Connector. This Voice Connector will be used as the call audio source.
```

Comment:
make it very clear that we mean the id, not the name.. eg 

(Optional) The Voice Connector *ID* (not Name) of an existing Amazon Chime SDK Voice Connector...